wordpress-mobile/WordPress-iOS
# Site Creation : Integration of Verticals prompt service

### Description
Fixes #10671

To test:

1. Checkout the branch and verify that existing tests pass
1. Because the branch for this PR was created from [this branch](https://github.com/wordpress-mobile/WordPress-iOS/pull/10786) (PR still outstanding), you should only need to review the net addition of three commits
1. Step through the Site Creation flow, and confirm that you see different Vertical prompts for varied Segments as shown in the screenshots below.

Update release notes:


### Caveats

- While the code can be reviewed as-is, I have affixed the `[Status] DO NOT MERGE` label as a signal that this should not be merged until the [companion PR](https://github.com/wordpress-mobile/WordPressKit-iOS/pull/67) from `WordPressKit` has been merged & versioned.
- I encountered a mismatch between the mocked Segment IDs and the values on the server, and had to update the simulated values here. This should not impact #10316.

### Screenshots

<img width="584" alt="10671_en_blog" src="https://user-images.githubusercontent.com/221062/51006637-15ffab00-14f9-11e9-9e6c-5ec2cd4e9530.png">
<img width="584" alt="10671_en_pro" src="https://user-images.githubusercontent.com/221062/51006638-15ffab00-14f9-11e9-9777-9bfa976f685f.png">
